alphazero/_game.py

```diff
@@ -262,10 +263,7 @@ def is_game_over(board) -> bool:
         just return a boolean True/False.
         """
 
-        if any(TicTacToe.get_game_status(board)):
-            return True
-        # implicit else
-        return False
+        return not TicTacToe.result(board) is None
```

Comment:
definitely feel like this line could be clarified a lot more... maybe capture the result separately, then eval if none, and then return? I find this line hard to read

Reply:
Not sure what you mean, I'll leave that for you to change